docs/astro/src/content/docs/reference/global-namespaces/platform.mdx

```diff
@@ -0,0 +1,36 @@
+---
+<!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
+title: Platform
+description: Platform Namespace
+---
+
+import SlintProperty  from '/src/components/SlintProperty.astro';
+import Link from '/src/components/Link.astro';
+
+The **Platform** namespace contains properties that help deal with platform specific differences.
+
+## Properties
+
+### os
+<SlintProperty propName="os" typeName="string">
+The name of the operating system. Possible values are:
+
+| Operating System | Value     |
+|------------------|-----------|
+| Windows          | `windows` |
+| macOS            | `macos`   |
+| Linux            | `linux`   |
+| Android          | `android` |
+| iOS              | `iOS`     |
```

Comment:
"iOs"  as a uppercase O, should macOs too? and Windows and Linux and Android?

Reply:
Oops, the values are intended to be lower-case. I'll fix that.